### PR TITLE
added mvp definition

### PR DESCRIPTION
**Pull request by TanyaPina**
## This PR fixes...

This PR adds the definition for "MVP" to the dictionary page.

## What I did...

- Added a definition to MVP
- Added the source URL of the definition
- Added the meaning of the term from the perspective of a software developer.

## How to test...

1. Navigate to the Netlify deploy preview here: https://deploy-preview-219--cherryontech.netlify.app/
2. Scroll down to the term "MVP" or enter the term "MVP" in the search bar. 

## I learned...

How a YAML file looks like!

### CODE REVIEW

**Review by shelleymcq — APPROVED** [review]
Looks great!

File: src/assets/content/dictionary/mvp.md
Comment (KatKowalik) [inline src/assets/content/dictionary/mvp.md:1]: Good job! I think there is a typo in the word "furtur" in the last sentence, it should be "further" :)

**Review by KatKowalik — CHANGES_REQUESTED** [review]
Good job! I think there is a typo in the word "furtur" in the last sentence, it should be "further" :) Besides that, everything looks good

File: src/assets/content/dictionary/mvp.md
Comment (TanyaPina) [reply src/assets/content/dictionary/mvp.md:1]: Nice catch, it has been fixed!

**Review by KatKowalik — APPROVED** [review]

**Comment by avinashi10** [comment]
Hey Tanya - the definition looks great! I would recommend slightly editing your Pull Request 'How to test...' section. If the tester goes to the live site they won't see the definition for MVP as your branch hasn't been merged yet. Instead instruct them on how to run your branch locally or use the netlify deploy preview above to test your changes.

**Review by YashsviG — APPROVED** [review]

**Comment by TanyaPina** [comment]
> Hey Tanya - the definition looks great! I would recommend slightly editing your Pull Request 'How to test...' section. If the tester goes to the live site they won't see the definition for MVP as your branch hasn't been merged yet. Instead instruct them on how to run your branch locally or use the netlify deploy preview above to test your changes.

Great point @avinashi10! I've updated the link to be the Netlify deploy preview.

**Pull request merged**